rholang/src/main/scala/coop/rchain/rholang/interpreter/normalize.scala

```diff
@@ -621,53 +628,58 @@ object ProcNormalizeMatcher {
       case p: PExprs =>
         normalizeMatch[M](p.proc_, input)
 
-      case p: PSend => {
-        for {
-          nameMatchResult <- NameNormalizeMatcher.normalizeMatch[M](
-                              p.name_,
-                              NameVisitInputs(input.env, input.knownFree)
-                            )
-          initAcc = (
-            Vector[Par](),
-            ProcVisitInputs(VectorPar(), input.env, nameMatchResult.knownFree),
-            BitSet(),
-            false
-          )
-          dataResults <- p.listproc_.toList.reverse.foldM(initAcc)(
-                          (acc, e) =>
-                            normalizeMatch[M](e, acc._2).map(
-                              procMatchResult =>
-                                (
-                                  procMatchResult.par +: acc._1,
-                                  ProcVisitInputs(
-                                    VectorPar(),
-                                    input.env,
-                                    procMatchResult.knownFree
-                                  ),
-                                  acc._3 | procMatchResult.par.locallyFree,
-                                  acc._4 || procMatchResult.par.connectiveUsed
-                                )
-                            )
-                        )
-          persistent = p.send_ match {
-            case _: SendSingle   => false
-            case _: SendMultiple => true
-          }
-        } yield
-          ProcVisitOutputs(
-            input.par.prepend(
-              Send(
-                nameMatchResult.chan,
-                dataResults._1,
-                persistent,
-                ChannelLocallyFree
-                  .locallyFree(nameMatchResult.chan, input.env.depth) | dataResults._3,
-                ChannelLocallyFree.connectiveUsed(nameMatchResult.chan) || dataResults._4
-              )
-            ),
-            dataResults._2.knownFree
-          )
-      }
+      case p: PSend =>
+        if (containsConnective(p.listproc_)) {
+          sync.raiseError(SyntaxError(s"Patterns are not allowed in sends."))
```

Comment:
I think that the message does not match the predicate. We already have:
1. Sending a free variable:
```
Top level free variables are not allowed: y at 1:15
```
2. Sending a wildcard:
```
Top level wildcards are not allowed: _ (wildcard) at 1:15
```
now you're adding a logical connective check. Please align the message so it fits the pattern we already have (as above).